task-1/src/Triangle.ts

```diff
@@ -0,0 +1,40 @@
+import { Shape } from './Shape';
+import { Point } from './Point';
+
+export class Triangle extends Shape {
+    constructor(point1: Point, point2: Point, point3: Point, color?: string, filled?: boolean) {
+        super([point1, point2, point3], color, filled);
+        this.points = [point1, point2, point3];
```

Comment:
This code is redundant because this assignment happened inside of the Shape constructor